initramfs/init.sh

```diff
@@ -169,7 +165,7 @@ probe_module()
 enable_usb_storage_device()
 {
     echo "Enable usb storage device driver."
-    modules="usbmisc_imx usb_otg_fsm ci_hdrc phy_mxs_usb ci_hdrc_imx"
+    modules="usb251xb"
```

Comment:
What are these other modules, and do we not need them?

Reply:
see the above comment to get the USB working.